compatibility/test.sh

```diff
@@ -69,4 +109,8 @@ bazel test //... \
   --test_env "POSTGRESQL_HOST=${POSTGRESQL_HOST}" \
   --test_env "POSTGRESQL_PORT=${POSTGRESQL_PORT}" \
   --test_env "POSTGRESQL_USERNAME=${POSTGRESQL_USERNAME}" \
+  --test_env "ORACLE_PORT=${ORACLE_PORT}" \
+  --test_env "ORACLE_USERNAME=${ORACLE_USERNAME}" \
+  --test_env "ORACLE_PWD=${ORACLE_PWD}" \
+  --test_env "DOCKER_PATH=${DOCKER_PATH}"
```

Comment:
Passing docker path to the test, so that the test uses the same docker instance that was used to start the database.